docs/advanced-configuration.md

```diff
@@ -379,14 +379,19 @@ agent:
 ### Known issues
 
 Kube Proxy
-* https://github.com/kubernetes/kops/issues/6472
-  * Problem
-    * When using a kops created Kubernetes cluster, a network connectivity issue has been reported that prevents proxy
-      metrics from being collected.
+* `10249: connect: connection refused`
```

Comment:
Do you know what logs/signals will persist to demonstrate configuration or connectivity issues with the change in log level? Does this error get reported on failed receiver startup?

Reply:
After this change, users can view failure information and related stack traces in the collector agent logs. Prior to the change, these failures were logged at the error level, but they will now be logged at the debug level. It's important to note that these failures do not cause the collector to fail to start; the collector continues to operate normally. However, the failure is logged at regular intervals, which is determined by the configured interval. By default, the metric collection interval is set to 10 seconds, so this failure will be logged every 10 seconds.

Reply:
The agent log level isn't set though right (so it'd be info)? I mean without setting debug service telemetry logging.

Reply:
That is correct, the agent will still use the default log level info. A user would have to update the [service telemetry logging level to debug](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md) in order to view these types of failures after this change.

Debugging this type of failure using debug logs shouldn't be unusual to users who are familiar with how to debug Prometheus issues.

Reply:
Thanks, I'm referring to this scenario as being a potential issue. In order to reduce chattiness for some unsupported services on some deployments, we are now defaulting to silencing errors that could mask larger problems. My initial thought would be to leave the level at error for the api server or similar, or even add a new value to set the level for these components that is opt-in to debug.

Reply:
We find ourselves in a situation where we need to balance the issue of "noisy logs in unsupported environments" with "inadequate log noise in supported environments."

To address this trade-off, I suggest taking a minimalist approach that can offer significant benefits. Based on the previous comments, I propose incorporating the scrapeFailureLogLevel: debug only for the kube-proxy receiver when the distribution is set to [""](https://github.com/signalfx/splunk-otel-collector-chart/blob/715d20e8f541187f0d8733dfabc935328a8c5c70/helm-charts/splunk-otel-collector/values.yaml#L181).

I plan to implement these changes soon, unless someone else presents an alternative approach they would like to suggest.

Reply:
Thanks Josh, focusing on unsupported environments makes sense. Let's start here and make sure the log level is a note in product docs too (customers should be aware of how to update the values.yaml file to override this). While we have this implemented for kube-proxy at this point, do we have similar pain points through issues for other control plane components as well?

Reply:
@aunshc The other control plane components have barely had barely any of the same pain point issues (when it comes to custom k8s distribution implementation) compared to kube poxy . This could change in the future (seems unlikely at this time). We can focus only on addressing the kube proxy pain point for now.